src/menu/tray.js

```diff
@@ -60,7 +62,16 @@ export class Tray extends EventEmitter {
     this.#trayMenu.setToolTip('Rancher Desktop');
 
     // Discover k8s contexts
-    this.updateContexts();
+    try {
+      this.updateContexts();
+    } catch (err) {
+      if (err instanceof TypeError &&
```

Comment:
There should be an `else` that re-throws the error, probably?

Reply:
true

Reply:
On second thought, the error should also be presented in a dialog box, and not thrown. If it's thrown, the app quietly fails to start.